gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -193,7 +197,11 @@ <h3 class="mdc-layout-grid__cell--span-3">
           <div class="mdc-layout-grid__cell--span-9">
             <ul class="severity">
               {% for item in affected.severity -%}
-              <li>{{ item.type }} - {{ item.score }}</li>
+              <li>
+                {{ item.type }} - {{ item | display_severity_rating }} - {{ item.score }}
```

Comment:
Question: Do we want to display the score using coloured tags here? e.g.:

 {7.8 (High)-coloured tag} - CVSS_V3 - CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H [CVSS Calculator](https://www.first.org/cvss/calculator/3.1#CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H)

Reply:
Yes, I think that'll be great!

Reply:
Updated:

<img width="978" alt="Screenshot 2024-06-27 at 7 02 54 PM" src="https://github.com/google/osv.dev/assets/73332835/20c46386-8a02-43b1-90e3-8f5b77c1136c">

Reply:
hold on, I discovered a bug there. I'll fix it tomorrow.

Reply:
It should be fixed now. Added the tag to severity on affected section too. `BIT-mediawiki-2020-10534`

<img width="1294" alt="Screenshot 2024-06-28 at 10 25 49 AM" src="https://github.com/google/osv.dev/assets/73332835/00d38e8d-f4be-4c77-b1f5-31a18089ef1a">

Reply:
Question: Are there cases where the severity score of an affected package differs from the severity score of its vulnerability? If the severity scores are identical, it may not be necessary to duplicate the severity within each affected package.

Reply:
Yep, I reckon it's possible to have different severity types with different score. I recall I tested with some samples when working on the severity tag in the list page. but couldn't find it now.

Reply:
Oh yeah, some vulnerabilities define [severity for each specific package](https://ossf.github.io/osv-schema/#affectedseverity-field).